### DISCUSSION TITLE
Visual Studio Community 2022

### DISCUSSION THREAD

**Discussion by LuizH1359** [balta-io/forum#253] [Tira Dúvidas] [answered]
Bom dia, estou utilizando o visual Studio Community 2022 para realizar o curso de Domínios Ricos, porém notei que ele não está sublinhando os erros como o Visual Studio Code, como posso habilita-lo para sublinhar erros como no code?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @LuizH1359, como vai?
Tem algum erro no Event Viewer do Windows apontando o Visual Studio? Normalmente quando alguma situação assim acontece, é algum bug...

Sugestão 1:
- Fechar o Visual Studio
- Limpar o cache da máquin
- Reiniciar

Sugestão 2:
- Repetir a sugestão 1
- Remover o Visual Studio
- Instalar novamente
